### ISSUE TITLE
meaningful link content

### ISSUE THREAD

**Issue by lassediercks** [filtercake/filtercake.github.io#12]
[according to the wcag](https://www.w3.org/TR/WCAG20-TECHS/H30.html) anchorlinks should be provided with purpose in the content of the link

[in this section](https://github.com/filtercake/filtercake.github.io/blob/master/index.html#L328) you could extend the link to `rant on twitter` or `push to github`.

The same applies to the navgiation where it would provide more meaning if you read a link out loud if the links content would be: `created the koma (korn mate) drink` or `organisised ux camp hamburg`

**Comment by filtercake** [comment]
> extend the link to rant on twitter or push to github

👍

> The same applies to the navgiation

I see, for now this current version is intended to unblock from too specific wording/crediting :)

will keep in mind for the future though!
